apps/angular/src/app/features/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,47 @@
+import { MatTableModule } from '@angular/material/table';
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { FetchAnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Anime } from '@js-camp/core/models/anime';
+import { Observable } from 'rxjs';
+import { CommonModule, NgOptimizedImage, DatePipe } from '@angular/common';
+import { MatChipsModule } from '@angular/material/chips';
+
+import { EmptyPipe } from '../../../shared/pipes/empty.pipe';
+
+/** Anime table column names. */
+enum AnimeTableColumns {
+	Image = 'image',
+	TitleEng = 'English title',
+	TitleJpn = 'Japanese title',
+	StartDate = 'Start date',
+	Type = 'type',
+	Status = 'status',
+}
+
+/** Anime table component. */
+@Component({
+	selector: 'anime-table',
+	styleUrl: './anime-table.component.css',
+	templateUrl: './anime-table.component.html',
+	standalone: true,
+	imports: [MatTableModule, CommonModule, NgOptimizedImage, MatChipsModule, EmptyPipe, DatePipe],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimeTableComponent implements OnInit {
+
+	/** Anime table column names. */
+	protected readonly animeColumns = AnimeTableColumns;
+
+	/** An array that determines anime table columns order. */
+	protected readonly displayedColumns = Object.values(this.animeColumns);
+
+	/** Stream containing anime data from the server. */
+	protected animeData$!: Observable<Anime[]>;
+
+	public constructor(private fetchAnimeService: FetchAnimeService) {}
```

Comment:
```suggestion
  private readonly fetchAnimeService = inject(FetchAnimeService);
  
	public constructor() {}
```

Let's use inject to import services